jekyll/_sass/_sidebar.scss

```diff
@@ -15,8 +16,7 @@ $sidebar-spacing-base: 5px;
   height: 92vh;
```

Comment:
This 92vh value is very approximate. 

Setting a `top` value equal to the height of the top nav, and `bottom: 0` instead of height would always be correct. (or `height: calc(100vh - #{$nav-height})` ... except the nav height isn't consistent across screensizes.

Reply:
I agree this is a poor way to specify this element, but its existence never bit us before. I'll update it to the rule you suggested. I'd prefer this to specifying the top value trying to match the nav height, because if the height calculation gets off in the future it's less catastrophic than an incorrect top value.